src/promptflow-devkit/promptflow/batch/_batch_engine.py

```diff
@@ -151,6 +151,7 @@ def __init__(
         self._storage = storage if storage else DefaultRunStorage(base_dir=self._working_dir)
         self._kwargs = kwargs
 
+        self._batch_use_async = kwargs.get("batch_use_async", False)
```

Comment:
Where would the "_batch_use_async" be used? I only see some "not" condition checks.

Reply:
It's used by the evaluate API based on the evaluate callable type (sync or async). Please refer to Ln38 in proxy_client.py. It's propagated from pf.run to the batch engine through the kwargs.